tests/test_pools.py

```diff
@@ -10,7 +11,7 @@
 
 # Min liquidity capitalization in USDC for a pool to be considered
 # in tests. Arbitrarily chosen as to avoid flakiness.
-min_pool_liquidity_cap_usdc = 50_000
+min_pool_liquidity_cap_usdc = 100_000
 num_all_pools_expected = 1500
```

Comment:
Core change: 1.

Increasing the filter. We do not need to test this against every pool. As long as ~20 pools are correct, we can extrapolate with confidence